the-collab-lab/tcl-27-smart-shopping-list
# This is for you Graham!! WootWoot!!

## Description

The app looks great!

![OMG-owl](https://media.giphy.com/media/vQqeT3AYg8S5O/giphy.gif)

## Related Issue

Closes #13 

## [Acceptance Criteria](https://github.com/the-collab-lab/tcl-27-smart-shopping-list/issues/13)

As a user, I want the the app to look professional and welcoming.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="1680" alt="Screen Shot 2021-09-02 at 8 50 44 PM" src="https://user-images.githubusercontent.com/33358502/131934387-26437e84-0b92-4c69-8b02-ebba8a578499.png">

<img width="1677" alt="Screen Shot 2021-09-02 at 8 50 34 PM" src="https://user-images.githubusercontent.com/33358502/131934461-943070fb-5fd7-4843-881d-030e43c207b0.png">

<img width="1680" alt="Screen Shot 2021-09-02 at 8 50 24 PM" src="https://user-images.githubusercontent.com/33358502/131934469-1f312e95-c51e-4548-b126-30dc35b7d72a.png">


### After

<img width="1455" alt="Screen Shot 2021-09-02 at 7 48 44 PM" src="https://user-images.githubusercontent.com/33358502/131934485-aae4c14d-5e02-4a82-baa2-f14f03f4a41c.png">

<img width="1936" alt="Screen Shot 2021-09-02 at 7 49 09 PM" src="https://user-images.githubusercontent.com/33358502/131934495-a92810ba-84ae-4431-bd5c-9bec81df1690.png">

<img width="1904" alt="Screen Shot 2021-09-02 at 7 51 41 PM" src="https://user-images.githubusercontent.com/33358502/131934505-400b8608-c6bf-4e75-bdfc-23f88a787624.png">

<img width="1759" alt="Screen Shot 2021-09-02 at 7 49 35 PM" src="https://user-images.githubusercontent.com/33358502/131934520-9b5e99ec-30f5-447e-ab4d-64fe44221bb2.png">

![You-jelly](https://media4.giphy.com/media/O0qcxJrLC1wIw/giphy.gif)

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin yh-dp-sk-tc-style-woot-woot` and check that branch out with `git checkout yh-dp-sk-tc-style-woot-woot`
- In your terminal run `npm install` 
- Either use an existing user token ("noaa lares sense") or create a new list to log in.
- Browse through the app and try to add/delete items. Check out the design and features. Enjoy!!

![play-those-drums](https://media1.giphy.com/media/bKThMhaYM9EBO/giphy.gif)
